### DISCUSSION TITLE
Notice: User accounts now must be at least 24hrs old to post

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#50950] [Announcements]
Hi folks!

About a month ago, we experimented with turning on [Temporary Interaction Limits](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-organization) for github.com/community with the hopes that we would reduce spammy behavior in this space. [See our original post here](https://github.com/orgs/community/discussions/48334).

We're happy to announce, that experiment was a huge success!

## Spam and off-topic posts **are down nearly 70% overall** 

While, yes, there are still spammy posts, the reduction in junk posts has been dramatic. This is reducing the burden on our community managers and our community members.

In order to keep this space clean, we have decided to turn this one-month experiment into a **permanent setting**.

Additionally, while we were worried about new users feeling left out, it does appear that new users are simply searching the space and finding answers without much issue. We even actually saw an _increase_ in legitimate user engagement and contribution. 

## What this means

* All new users must wait until their account is at least 24hrs old to participate in the github.com/community Discussions space.
* Accounts that are at least 24hrs old will be able to post and interact as normal.
* If your account is less than 24hrs old and you have a question, please try using the Discussions search bar above to see if your question has already been asked or simply wait a day. We apologize if this causes any inconvenience.
* There will still be some spam (we're cooking up some more solutions)

If you'd like to learn more about implementing temporary interaction limits in your own orgs and repos, please read up on the official documentation [here](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-organization).

Best,
Your community moderation team

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
